src/models/AdminTask/WhereClauseBuilder/OfferExpirationDate.ts

```diff
@@ -19,16 +19,11 @@ export const OfferExpirationDateWhereClause = {
 
     return `
       (
-        "${Offer.tableName}"."${expirationDateProperty}" > '${new Date().toISOString()}'
-        OR
-        ( 
-          "${Offer.tableName}"."${expirationDateProperty}" IS NULL
+        NOT 
```

Comment:
agregaria un test de que los estados not approved no pueden tener expiration date distinta de null

Reply:
es decir, validacion a nivel modelo